src/features/dialog-manager/reducer.js

```diff
@@ -40,14 +42,40 @@ const initialState = {
     },
     characterCreation: false,
     character: {
-        characterName: '',
+        characterName: ' ',
         characterRace: 'Human',
         characterClass: 'Fighter',
     },
 };
 
 const dialogManagerReducer = (state = initialState, { type, payload }) => {
     const { abilities, abilities_minimum, character } = state;
+    // if in character creation mode
+    if (state.characterCreation && character.characterName !== ' ') {
+        resetAbilityScoreValues(abilities);
+        switch (character.characterRace) {
+            case 'Human':
+                abilities.strength = RACE_BONUS_STARTING_ABILITY_SCORE_VALUE;
+                abilities.intelligence = RACE_BONUS_STARTING_ABILITY_SCORE_VALUE;
+                abilities_minimum.min_strength = 2;
```

Comment:
It might be better to  have the `ABILITY_BONUS = 2` defined as a constant rather than the `RACE_BONUS_STARTING_ABILITY_SCORE_VALUE`.  

That way you could have `abilities_minimum.min_strength = ABILITY_BONUS;` and `abilities.strength = STARTING_ABILITY_SCORE_VALUE + ABILITY_BONUS;` (or `abilities.strength += ABILITY_BONUS;`)

Reply:
Will do, the minimum values are all manually set to 0 in the initial state, should we convert those to constants as well?

Reply:
I think 0's should be fine (unless we ever plan on changing that value).